Octokit/Models/Response/GitHubCommitFile.cs

```diff
@@ -86,6 +86,27 @@ public GitHubCommitFile(string filename, int additions, int deletions, int chang
         [Parameter(Key = "previous_filename")]
         public string PreviousFileName { get; protected set; }
 
+        /// <summary>
+        /// The SHA the file was commited in
+        /// </summary>
+        public string CommitSha
```

Comment:
This feels like it would be more appropriate as an extension or helper.  If possible, I'd like to keep composite and logic-filled properties out of the models.  Otherwise, we will have to develop a more complex / override-based generation method.

My preference would be that if we are going to provide this type of helper that it be hoisted out of the model itself or in the related [question](https://github.com/octokit/octokit.net/issues/2411) you offer it as a solution that could be implemented in the user's app itself.